src/status_im2/contexts/quo_preview/tags/number_tag.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im2.contexts.quo-preview.tags.number-tag
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:key     :type
+    :type    :select
+    :options [{:key   :rounded
+               :value "Rounded"}
+              {:key   :squared
+               :value "Squared"}]}
+   {:key  :number
+    :type :text}
+   {:key     :size-key
+    :type    :select
+    :options [{:key   :size/s-32
+               :value "32"}
+              {:key   :size/s-24
+               :value "24"}
+              {:key   :size/s-20
+               :value "20"}
+              {:key   :size/s-16
+               :value "16"}
+              {:key   :size/s-14
+               :value "14"}]}
+   {:key  :blur?
+    :type :boolean}])
+
+(defn preview
+  []
+  (let [state (reagent/atom {:type     :squared
+                             :number   "148"
+                             :size-key :size/s-32
+                             :blur?    false})]
+    (fn []
+      [preview/preview-container {:state state :descriptor descriptor}
+       [rn/view
```

Comment:
we need this outer view now? 🤔

Reply:
Thanks for you review @J-Son89!

I could remove it, but it wouldn't be centered anymore - it would be in the top left corner, under the `Blur?` label. WDYT?

Reply:
nah if it's needed for the ui then it's fine. I was just wondering if it's better in the preview-container functionality to handle that directly.

Reply:
sounds good, left as-is